### PR TITLE
Threshold fraud

### PR DESCRIPTION
**Pull request by jdemonasterio**
A blogpost on the methodology we internally use to decide, for each app_id, which is the timedelta threshold used to filter installs that are coming in too late. 

The blog describes the problem, the description of the data related to the TimeDelta Metric, the different approximations, per app, for these probabilistic distributions and the algorithm used to create a threshold per app.

All edits are welcome :)

### CODE REVIEW

**Comment by jdemonasterio** [comment]
Tagging @mbonamico @juanpampliega @GervasioCalderon to the conversation.
Thanks guys.

**Comment by sebamontini** [comment]
maybe asking ceci2 to "draw" some custom graphics instead of using the bokeh default ones?

File: _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+![Fraud App Threshold Example 2 ]({{ site.url }}/assets/images/fraud/hist-app_threshold_value2.png "For this case we se a similar behavior but we find the decay to be much faster. The tail is much shorter than before and the threshold is set slightly over three hours. This is significantly different to the case before."){: .center-image }
+
+For this case we se a similar behavior but we find the decay to be much faster. The tail is much shorter than before and the threshold is set slightly over three hours. This is significantly different to the case before.
+
+Finally, to build an install threshold for today's date. We take a timespan of data and record all of the the thresholds calculated from those cases which were were not regarded as coming from fraudulent behavior. From this data, we calculate for each app the median threshold and this value will be our cut. The median statistic has the advantage of being robust to outliers and this is important for cases where fraudulent behavior has slipped through as "normal". 
+
+
+#Conclusion and Drawbacks
```
Comment (juanpampliega) [inline _src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown]: I would remove `Drawbacks` from the title.

**Comment by jdemonasterio** [comment]
@sebamontini agree it would be nicer. However I don't know if there are any time requirements though on this post. @mbonamico 

ps: it's `seaborn` not `bokeh` 😝

**Comment by mbonamico** [comment]
yesterday

**Comment by jdemonasterio** [comment]
> At least one approved review is required by reviewers with write access. 
Ok then @mbonamico @juanpampliega I don't have write access so I'll need an approved review by any of you. 👍

**Pull request merged**